src/Console/DeployAgentCommand.cs

```diff
@@ -41,7 +41,8 @@ public async Task DeployHandler(DeployParameters parameters)
             NewEnvironmentHandler newEnvironmentHandler = ServiceLocator.Get<NewEnvironmentHandler>();
             await newEnvironmentHandler.Run(parameters.Name, parameters.Parameters, (EnvironmentType)Enum.Parse(typeof(EnvironmentType), parameters.Type), parameters.Site, null, null,
                 (DeploymentTarget)Enum.Parse(typeof(DeploymentTarget), parameters.Target), parameters.Output,
-                parameters.ReplaceTokens, parameters.Interactive, parameters.CustomerInfrastructureName, parameters.Description, false, true, parameters.DeploymentTimeoutMinutes);
+                parameters.ReplaceTokens, parameters.Interactive, parameters.CustomerInfrastructureName, parameters.Description, false, true, parameters.DeploymentTimeoutMinutes,
+                false, false);

```

Comment:
postponing to 2nd phase

Reply:
Currently the flags require the terminateOtherVersions flag to be active. Since here this flag is always false, these flags are not supported as well. Moved to 2nd phase of the WI since it makes sense to make these available to the agent as well